src/test/resources/testbib/complex.bib

```diff
@@ -296,8 +296,6 @@ @Comment{jabref-meta:
 1 KeywordGroup:DynamicGroup\;0\;author\;Churchill\;0\;0\;;
 }
 
-@Comment{jabref-meta: groupsversion:3;}
```

Comment:
Is it ok to just remove that? Wouldn't it be better to have it in and verify that it is just ignored by the new parser?